packages/back-end/src/util/secrets.ts

```diff
@@ -214,3 +214,18 @@ const getTrustProxyConfig = (): boolean | string | number => {
 };
 
 export const EXPRESS_TRUST_PROXY_OPTS = getTrustProxyConfig();
+
+export const AGENDA_DEFAULT_LOCK_LIMIT =
+  parseInt(process.env.AGENDA_DEFAULT_LOCK_LIMIT || "") || 5;
+
+export const AGENDA_LOCK_LIMIT =
+  parseInt(process.env.AGENDA_LOCK_LIMIT || "") || 5;
```

Comment:
Looking in the Agenda source code, it looks like the default lockLimit value is `0`, not `5`.